<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_102">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_103">
          <item>
           <widget class="QLabel" name="label_92">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No de lote:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputBatchMateriaPrima">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_104">
          <item>
           <widget class="QLabel" name="label_93">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Código:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputCodeMateriaPrima">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_108">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_109">
          <item>
           <widget class="QLabel" name="label_96">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Peso neto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputPesoNeto">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_110">
          <item>
           <widget class="QLabel" name="label_97">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Peso bruto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputPesoBruto">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="imageRot_6">
        <property name="minimumSize">
         <size>
          <width>231</width>
          <height>81</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>231</width>
          <height>81</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../media/logo-byspro.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_115">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_116">
          <item>
           <widget class="QLabel" name="label_101">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Pesado por:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputSignature">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_117">
          <item>
           <widget class="QLabel" name="label_102">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Revisado por:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_50">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_88">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black; </string>
          </property>
          <property name="text">
           <string>PN-MA-01-PR-01-FO-01</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_89">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Rótulo de pesaje</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_100">
          <item>
           <widget class="QLabel" name="label_90">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Fecha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputDate">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QCheckBox" name="checkEntregar">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>📦</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>🗑</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QCheckBox" name="checkSyncDate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>📅</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_105">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_106">
          <item>
           <widget class="QLabel" name="label_94">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No de control:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputNumControl">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_107">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_95">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Tara:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputTara">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_112">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_113">
          <item>
           <widget class="QLabel" name="label_99">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Código de producto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputCode">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_114">
          <item>
           <widget class="QLabel" name="label_100">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>No. de lote:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputBatch">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_101">
        <item>
         <widget class="QLabel" name="label_91">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Materia prima:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputMateriaPrima">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_111">
        <item>
         <widget class="QLabel" name="label_98">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nombre del producto:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputName">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
border-bottom: 1px solid black; background-color: transparent;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
